### ISSUE TITLE
Folders remain selected

### ISSUE THREAD

**Issue by kearaa** [mdelobelle/metadatamenu#695]
After right-clicking a folder, folders remain outlined as if they are selected even after clicking out of them. They stay this way until Obsidian is reloaded. 

I tested it with safe mode, and this did not happen. I disabled and re-enabled plugins to find out which one was causing this to happen. When I enable metadata menu, the problem comes back. 

in Obsidian v1.6.3

![CleanShot 2024-06-17 at 12 25 58@2x](https://github.com/mdelobelle/metadatamenu/assets/68878233/28a47506-9017-40c6-99a4-47c0775e7043)

**Comment by jqml** [comment]
same issue too

**Comment by merlinuwe** [comment]
A friend of mine has the same issue. Read more here:

https://www.reddit.com/r/ObsidianMD/s/vn0XO45qkV

He has only 6 plugins installed, so if one of them is in conflict with yours, it will be easy to find.

**Comment by MelodyDuplaix** [comment]
I have the same issue too.

**Comment by ChristinWhite** [comment]
@merlinuwe I don't think it's a conflict, I disabled everything else trying to figure out where it was coming from when I first noticed it, it's probably something that could be fixed with a CSS snippet, but I haven't had a chance to try it yet.

**Comment by ChristinWhite** [comment]
I tweaked my CSS today to deal with this issue and this seems to be a safe hack to prevent a persistent border. Just drop it into a [CSS snippet](https://help.obsidian.md/Extending+Obsidian/CSS+snippets):

```css
.tree-item-self.has-active-menu {
	box-shadow: none !important
}
```

This will remove the border even when actually right-clicking on a tree item. However, it's still pretty obvious what you clicked on thanks to the hover styles.

![CleanShot 2024-10-05 at 17 31 54](https://github.com/user-attachments/assets/f4266bf3-7bae-41ea-a782-af5692c29f56)

I also right-clicked `Areas` and `Neovim Config` before `Launchpad OS`, while they both keep the `.has-active-menu` class the visual attributes associated with it are gone so you won't notice.

Note that I've only tested this with the default theme and [Minimal](https://github.com/kepano/obsidian-minimal), other themes might apply other styles that will need to be negated. The selector shouldn't need to change.

**Comment by xPeyotii** [comment]
Still an issue sadly

**Comment by luckymouse2332** [comment]
I'm running into the same problem. 
@ChristinWhite, thank you for the CSS workaround, but it's a bit of a 'scorched earth' solution since it hides the focus state completely. I'd prefer a way to deselect the folder (via `ESC` or clicking whitespace) like before. 
It would be great if the author could look into this!

**Comment by ChristinWhite** [comment]
@luckymouse2332 Yeah, it's not ideal; hopefully it's not a large change, but I'm not really sure and have largely moved away from MetadataMenu at this point.
